### ISSUE TITLE
"Getting started" documentation

### ISSUE THREAD

**Issue by EmmanuelMess** [MinecraftForge/MinecraftForge#6596]
In https://mcforge.readthedocs.io/en/latest/forgedev/ at 
Setting Up the Environment > IntelliJ IDEA > 4 it says:

> Open the “forge” project tree, select “Tasks”, then “forgegradle” and right click the “Create Forge [setup]” option

There is no forgegradle not setup anymore.

Possible fix: replace with "double ctrl (run anything) > type 'gradle clean build'".

**Comment by EmmanuelMess** [comment]
Wrong repo, sorry!

**Comment by huntert1004** [comment]
Having the same issue, what repo worked for you?

**Comment by EmmanuelMess** [comment]
@huntert1004 the documentation should be updated by now, but if not check the last version at https://github.com/MinecraftForge/Documentation or https://mcforge.readthedocs.io/en/1.14.x/forgedev/

TL;DR: Run "gradle setup" in IntelliJ 2019+
